internal/characters/tighnari/skill.go

```diff
@@ -41,28 +46,28 @@ func (c *char) Skill(p map[string]int) action.ActionInfo {
 	c.Core.QueueAttack(
 		ai,
 		combat.NewCircleHit(c.Core.Combat.Player(), 2, false, combat.TargettableEnemy),
-		0,
-		skillHitmark,
+		skillRelease,
+		skillRelease+travel,
 	)
 
 	var count float64 = 3
 	if c.Core.Rand.Float64() < 0.5 {
 		count++
 	}
-	c.Core.QueueParticle("tighnari", count, attributes.Dendro, skillHitmark+c.ParticleDelay)
+	c.Core.QueueParticle("tighnari", count, attributes.Dendro, skillRelease+travel+c.ParticleDelay)
 	c.SetCDWithDelay(action.ActionSkill, 12*60, 13)
 
-	c.AddStatus(vijnanasuffusionStatus, 12*60, false)
+	c.Core.Tasks.Add(func() { c.AddStatus(vijnanasuffusionStatus, 12*60, false) }, 13)
 	c.SetTag(wreatharrows, 3)
 
 	if c.Base.Cons >= 2 {
-		c.QueueCharTask(c.c2, skillHitmark+1)
+		c.QueueCharTask(func() { c.c2(travel) }, skillRelease+travel+1)
```

Comment:
you don't need to pass a param to c2, because the task pops out right on field start you can just make the c2 last for 14*60. also make the delay skillRelease+travel, we needed the +1 previously because of snapshot stuff